tests/scenarios/transform-complex-txn.spec.ts

```diff
@@ -273,15 +273,23 @@ describe('Complex transaction with many nested inner transactions', () => {
     expect(algodTxns.blockMetadata).toMatchInlineSnapshot(`
       [
         {
+          "currentProtocol": "https://github.com/algorandfoundation/specs/tree/abd3d4823c6f77349fc04c3af7b1e99fe4df699f",
+          "feeSink": "Y76M3MSY6DKBRHBL7C3NNDXGS5IIMQVQVUAB6MP4XEMMGVF2QWNPL226CA",
           "fullTransactionCount": 171,
           "genesisHash": "wGHE2Pwdvd7S12BL5FaOP20EGYesN73ktiC1qzkkit8=",
           "genesisId": "mainnet-v1.0",
           "hash": "CDVL4HL5XDN7DPYAHITVHKVZSX4KNRL65XBOCSQTBNDOFMHN7JBA",
           "parentTransactionCount": 55,
           "previousBlockHash": "6ELYXRVDR7DGXCVYC7AAVWLFIUNPLXT5QTNL2NL7ZQL3O66FMFWQ",
```

Comment:
As we chatted about, this value would be represented as `blk-6ELYXRVDR7DGXCVYC7AAVWLFIUNPLXT5QTNL2NL7ZQL3O66FMFWQ` in algod and `8ReLxqOPxmuKuBfACtllRRr13n2E2r01f8wXt3vFYW0=` in indexer. Right now we don't represent either an algod or indexer compatible value (it's very close to algod though).

Reply:
Yeah I remember looking at that, it took me ages to get the hash correct.

This felt like the most correct way to represent it, similar to a transaction ID or account address

Reply:
(The way I saw it, `blk-` was just a namespace qualifier and indexer showed the same data but converted to binary and shown in base64, which is inconsistent with the way the hash is shown for a transaction or account.)

Reply:
Side-note: the lack of consistency showed to me that block metadata was rarely used in the ecosystem (makes sense it's pretty low level) and so I tried to choose the most sane seeming approach.